contrib/com.syslog-ng.syslog-ng.plist

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<!--
+  launchd job for macOS that runs in the background upon system start.
+  
+  Tested environment: 
+    - macOS Catalina 10.15.3 
+    - syslog-ng 3.25
+    
+  Location to be stored: /Library/LaunchDaemons
+  Permissions: 644 root:wheel
+  If you need to manually start/stop the job:
+    - Start command: sudo launchctl load /Library/LaunchDaemons/com.syslog-ng.syslog-ng.plist
+    - Stop command: sudo launchctl unload /Library/LaunchDaemons/com.syslog-ng.syslog-ng.plist
+  
+  Author: Roberto Meléndez <@rcmelendez on LinkedIn & Medium>
+  Created: March 11, 2020
+  Modified: March 15, 2020
+-->
+<dict>
+	<key>Label</key>
+	<string>com.syslog-ng.syslog-ng</string>
+	<key>ProgramArguments</key>
+	<array>
+		<string>/usr/local/sbin/syslog-ng</string>
+		<string>-F</string>
+		<string>-p</string>
+    		<string>/var/run/syslogd.pid</string>
```

Comment:
I've just noticed that this path conflicts with the macOS syslogd's PID file path.

`/var/run/syslog-ng.pid` would be better.

Reply:
I saw that on Linux syslog-ng uses `/var/run/syslogd.pid`, that's why I chose that name and path. Also, macOS uses `/var/run/syslog.pid` for their syslog daemon. I have both `syslog.pid` and `syslogd.pid` working fine on my Mac. 

To avoid any confusion, I'll follow your advice :)

Reply:
Thanks. You're right, macOS uses `syslog.pid`, so `syslogd.pid` could actually work.
But yeah, `syslog-ng.pid` will avoid any confusion.